architecture/core_concepts/routes.adoc

```diff
@@ -213,6 +213,7 @@ variables on the *deployment config* for the router to alter its configuration.
 |`*ROUTER_SERVICE_HTTPS_PORT*` | 443 | Port to listen for HTTPS requests.
 |`*ROUTER_SERVICE_HTTP_PORT*` | 80 | Port to listen for HTTP requests.
 |`*ROUTER_SERVICE_NAME*` | public | The name that the router will identify itself with in route statuses.
+|`*ROUTER_CANONICAL_HOSTNAME*` | | The (optional) hostname of the router that is shown in the in route statuses.
```

Comment:
@pecameron Can I ask: "that is shown in the in route statuses".

Should that be "that is shown in the route status" or should it have markup (is it some sort of parameter or variable) like "that is shown in the `IN_ROUTE` statuses"

?